packages/shared/src/components/MainLayout.tsx

```diff
@@ -99,7 +99,7 @@ export default function MainLayout({
   });
   const [isDismissed, setIsDismissed, isFetched] = usePersistentContext(
     generateStorageKey(StorageTopic.Onboarding, 'wall_dismissed'),
-    false,
+    true,
```

Comment:
This sets the default to be dismissed?
How does it ever show to people then?

Reply:
Now I understand why it doesn't show anymore. Brain fart moments 😂